<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>display</class>
 <widget class="QMainWindow" name="display">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>489</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>534</width>
    <height>489</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>534</width>
    <height>489</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Results</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(52, 101, 164);
font: italic 17pt &quot;Century Schoolbook L&quot;;</string>
    </property>
    <property name="text">
     <string>CGPA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(52, 101, 164);
font: 57 23pt &quot;Century Schoolbook L&quot;;</string>
    </property>
    <property name="text">
     <string>Grade Point Average</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(52, 101, 164);
font: italic 17pt &quot;Century Schoolbook L&quot;;</string>
    </property>
    <property name="text">
     <string>GPA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(52, 101, 164);
font: italic 17pt &quot;Century Schoolbook L&quot;;</string>
    </property>
    <property name="text">
     <string>TOTAL SCORE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(52, 101, 164);
font: italic 17pt &quot;Century Schoolbook L&quot;;</string>
    </property>
    <property name="text">
     <string>TOTAL UNITS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(52, 101, 164);
font: italic 17pt &quot;Century Schoolbook L&quot;;</string>
    </property>
    <property name="text">
     <string>CO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(52, 101, 164);
font: italic 17pt &quot;Century Schoolbook L&quot;;</string>
    </property>
    <property name="text">
     <string>UNIT</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="GPAlineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 20pt &quot;Century Schoolbook L&quot;;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="CGPAlineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>241</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>241</width>
      <height>41</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 20pt &quot;Century Schoolbook L&quot;;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="UNITlineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 20pt &quot;Century Schoolbook L&quot;;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="TSlineEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 20pt &quot;Century Schoolbook L&quot;;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="TUTlineEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>300</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 20pt &quot;Century Schoolbook L&quot;;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="COlineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>350</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 20pt &quot;Century Schoolbook L&quot;;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="continue_btn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>410</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(92, 53, 102);
font: 75 14pt &quot;Century Schoolbook L&quot;;</string>
    </property>
    <property name="text">
     <string>Continue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(92, 53, 102);
font: 75 14pt &quot;Century Schoolbook L&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
